docs/manifest/equivalentaddins.md

```diff
@@ -36,6 +39,41 @@ For more information, see [Version overrides in the add-in only manifest](/offic
 
 - [EquivalentAddin](equivalentaddin.md)
 
+## Can contain
+
+The **\<EquivalentAddins\>** element can contain the following child element.
+
+|Element|Content|Mail|TaskPane|
+|:-----|:-----:|:-----:|:-----:|
+|[Effect](#effect)|No|No|Yes|
+
+### Effect
+
+Specifies either that the COM add-in is disabled and hidden (instead of the Office Add-in) when they conflict, or specifies that the user chooses which to disable and hide. There are two possible values.
```

Comment:
```suggestion
Specifies either that the COM add-in is disabled and hidden (instead of the Office Web Add-in) when they conflict, or specifies that the user chooses which to disable and hide. There are two possible values.
```